packages/swing-store/src/streamStore.js

```diff
@@ -1,7 +1,7 @@
 // @ts-check
 import { assert, Fail, q } from '@agoric/assert';
 
-const STREAM_START = { itemCount: 0 };
+const STREAM_START = 0;
```

Comment:
oh one more nit: could you move the type definition of `StreamStore` and `StreamPosition` to be defined here in `streamStore.js`, and imported by `swingStore.js`, instead of the other way around? That feels cleaner, and will make my `getAllState` patch cleaner too

Reply:
actually nevermind, I'll do it in my `getAllState` patch, this PR is not really changing `streamStore.js` enough to justify it here